### PR TITLE
Add treetracker ios testing instructions

### LINKED ISSUE #43: Create a documentation for treetracker app testing for iOS
Create a iOS version of this doc: https://github.com/Greenstand/treetracker-android/wiki/Treetracker-App-Testing-Instructions

1. Request an email 'invitation' from iOS Team to the testing environment
2. Download the TestFlight app on an iPhone
3. Press the 'Redeem' button on the top right corner and enter the invitation code from the email
4. Download the corresponding build on the phone
5. Sign up and login to the Treetracker testing app
6. Check ability to change users
7. Use app to capture image and record (Note: The app has a block for GPS accuracy. The user should be able to get GPS reception, "add a tree." Take a photo and upload the image.)
    - time per capture
    - any bugs
    - any unclear UX/UI
8. Sync data
9. Confirm data reaches the cloud by:
    - checking that the images should appear on the map at test.treetracker.org 
    - logging into the Tree Tracker Admin URL, https://test-admin.treetracker.org/
      - username: admin
      - password: ZwovDoOxCo
10. You can submit any defects/suggestions here under the Issues tab. https://github.com/Greenstand/treetracker-ios
11. If all checks ok, please report to the iOS Team

### PR DESCRIPTION
**Pull request by CorrWu**
I added the treetracker ios testing instruction document to the treetracker development folder. This should resolve [issue 43](https://github.com/Greenstand/greenstand-documentation/issues/43).

### CODE REVIEW

File: treetracker/development/treetracker-ios-testing-instructions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+7. Use app to capture image and record (Note: The app has a block for GPS accuracy. The user should be able to get GPS reception, "add a tree." Take a photo and upload the image.)
+    * time per capture
+    * any bugs
+    * any unclear UX/UI
+8. Sync data
+9. Confirm data reaches the cloud by:
+    * checking that the images should appear on the map at test.treetracker.org
+    * logging into the Tree Tracker Admin URL, https://test-admin.treetracker.org/
+        * username: admin
+        * password: ZwovDoOxCo
```
Comment (pranavkparti) [inline treetracker/development/treetracker-ios-testing-instructions.md]: It's better not to put the credentials here. It should be available in the Slack channel. You can link to the Slack and mention the channel name.

Comment (CorrWu) [reply treetracker/development/treetracker-ios-testing-instructions.md]: Okay, I'll change this part. By the way, this is on the documentation for android testing instructions as well.

Comment (CorrWu) [reply treetracker/development/treetracker-ios-testing-instructions.md]: Just updated it.

Comment (pranavkparti) [reply treetracker/development/treetracker-ios-testing-instructions.md]: @CorrWu Thank you for making the changes. This PR will be on hold for a bit until Gitbook is synced properly and some other issues are resolved. I will merge it after that.

Comment (Davidezrajay) [reply treetracker/development/treetracker-ios-testing-instructions.md]: @pranavkparti what else do we need before we can merge this?

**Pull request merged**